packages/cms/store/resource.ts

```diff
@@ -99,6 +108,11 @@ export const useResourceStore = create<ResourceState & ResourceMethods>(
       return await window.Tensei.api.post(resource?.slugPlural!, resourceInput)
     },
 
+    async updateResource(resourceId: number, resourceInput: AbstractData) {
+      const { resource } = get()
+      return await window.Tensei.api.patch(`${resource?.slugPlural!}/${resourceId}`, resourceInput)
```

Comment:
No need to `return await`. We can just `return` and it'll still resolve the promise correctly.